pyconbalkan/core/templates/includes/2019_event_sidebar.html

```diff
@@ -31,6 +31,7 @@ <h1>#{{ request.conference.number }}</h1>
 	{% if request.conference.twitter %}<a href="{{ request.conference.twitter }}" class="twitter" target="_blank"></a>{% endif %}
     </div>
     <h3 class="supported-by">Supported by</h3>
-    <a href="https://www.python.org/psf/"><div><img class="psf-logo" src="{% static 'img/pfs-logo.png' %}"></div></a>
-    <div><a href="https://www.djangoproject.com/"><img class="django-logo" src="{% static 'img/django-logo.png' %}"></a></div>
+    {% for supp in supported %}
+    <a href="{{ supp.url }}"><div><img src="{{ MEDIA_URL }}{{ supp.logo.name }}"></div></a>
```

Comment:
Use this: https://docs.djangoproject.com/en/dev/ref/templates/builtins/#get-media-prefix

It's better :)